pkgs/top-level/all-packages.nix

```diff
@@ -13949,10 +13949,10 @@ in
   openvdb = callPackage ../development/libraries/openvdb {};
 
   inherit (callPackages ../development/libraries/libressl { })
-    libressl_2_9
-    libressl_3_0;
+    libressl_3_0
+    libressl_3_1;
 
-  libressl = libressl_3_0;
+  libressl = libressl_3_1;
```

Comment:
Also, I'd like to see a comment here, indicating future contributors, that `libressl` should always point to the latest version. See https://discourse.nixos.org/t/nixpkgs-policy-regarding-libraries-available-in-multiple-versions/7026 .

Reply:
I added a comment, with a link to that topic.